custom/project_scrum_agile_extended/models/project_scrum.py

```diff
@@ -0,0 +1,125 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import _, api, fields, models
+
+
+class ProjectScrumSprint(models.Model):
+    _inherit = "project.scrum.sprint"
+
+    @api.depends(
+        "product_backlog_ids",
+        "product_backlog_ids.expected_hours",
+        "product_backlog_ids.effective_hours",
+        "state",
+        "estimate_adjustment",
```

Comment:
add product_backlog_ids.stage_id,release_id.sprint_ids,release_id.sprint_ids.expected_hours in depends and remove state and estimate_adjustment